model/src/main/scala/org/zalando/nakadi/client/model/Model.scala

```diff
@@ -0,0 +1,192 @@
+package org.zalando.nakadi.client.model
+
+/**
+ * The Event definition will be externalized in future versions of this document.
+ * A basic payload of an Event. The actual schema is dependent on the information configured for the
+ * EventType, as is its enforcement (see POST /event-types). Setting of metadata properties are
+ * dependent on the configured enrichment as well.
+ * For explanation on default configurations of validation and enrichment, see documentation of
+ * `EventType.type`.
+ * For concrete examples of what will be enforced by Nakadi see the objects BusinessEvent and
+ * DataChangeEvent below.
+ * @param eventType
+ * @param additionalProperties Default value is true
+ * @param title
+ */
+case class Event(
+  eventType: EventType,
+  additionalProperties: Boolean,
```

Comment:
I think it is not really a property but rather an indicator from the API description which let you know if an event can have arbitrary additional properties